app/views/employments/EndEmploymentIrregularPaymentErrorView.scala.html

```diff
@@ -1,68 +1,94 @@
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
 
 @import includes.backLink
 @import includes.forms.submit
 @import includes.{cancelLink, ptaHeader}
-@import uk.gov.hmrc.play.views.html.helpers._
 @import uk.gov.hmrc.tai.forms.employments.IrregularPayFormData
 @import uk.gov.hmrc.tai.viewModels.employments.EmploymentViewModel
 @import includes.forms.inputRadioGroupStyling
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF, govukRadios : GovukRadios, govukErrorSummary: GovukErrorSummary)
 
 @(irregularPayForm: Form[IrregularPayFormData], model: EmploymentViewModel)(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
-
-@header = @{
-    ptaHeader(
-        formForErrorSummary = Some(irregularPayForm),
-        displayBackLink = true,
-        mainHeadingText = Messages("tai.irregular.heading", model.employerName),
-        preHeadingText = Messages("tai.irregular.preHeadingText"))}
-
 @mainTemplate(
     title = Messages("tai.irregular.preHeadingText"),
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = None) {
-
-    <p>
-        <span class="lede">@Messages("tai.irregular.para1", model.employerName)</span>
-    </p>
-    <p>@Messages("tai.irregular.para2", model.employerName)</p>
-    <p>@Messages("tai.irregular.para3")</p>
-
-    @form(action= controllers.employments.routes.EndEmploymentController.handleIrregularPaymentError()) {
-
-        @{inputRadioGroupStyling(
-            field=irregularPayForm("irregularPayDecision"),
-            radioOptions=Seq(("contactEmployer" -> Messages("tai.irregular.option1", model.employerName)), ("updateDetails" -> Messages("tai.irregular.option2")) ),
-            legend=Messages("tai.irregular.para4"),
-            legendHint=None,
-            ('_groupDivClass, "form-group" ),
-            ('_fieldsetAttributes, "id=firstPayChoice" ),
-            ('_legendClass, "section flush"),
-            ('_legendSpanClass, "bold-small")
-        )}
+    backLinkContent= Some(Messages("tai.back-link.upper")),
+    backLinkUrl=Some(controllers.employments.routes.EndEmploymentController.employmentUpdateRemoveDecision().url)) {
+
+@if(irregularPayForm.errors.nonEmpty){
```

Comment:
some time ago i added something to the ptaHeader so u just need to populate these two fields so the if isn't necessary :) 
govukErrorSummary: Option[GovukErrorSummary]
formForErrorSummary